Makefile

```diff
@@ -21,6 +21,7 @@ coverage:
 
 ## install - Install and vendor dependencies
 install:
+	brew install golangci-lint
```

Comment:
This needs to `exit 0` as the fallback incase it's already installed.

Reply:
Fixed